zellij-client/src/lib.rs

```diff
@@ -416,6 +419,14 @@ pub fn start_client(
                     .send(InputInstruction::SwitchToMode(input_mode))
                     .unwrap();
             },
+            ClientInstruction::TabNameList(tab_names) => {
```

Comment:
but here can't receive `ClientInstruction::TabNameList`,  @imsnif am i doing right?

Reply:
I didn't look at this very deeply, but I think this should be handled in the `cli_client` if you are sending this query from the CLI

Reply:
Got it